core/controllers/dashboard.py

```diff
@@ -204,6 +204,7 @@ def _round_average_ratings(rating):
                     'created_on': utils.get_time_in_millisecs(
                         collection_summary.collection_model_created_on),
                     'status': collection_summary.status,
+                    'node_count': collection_summary.node_count,
```

Comment:
Ah, I didn't realize you were going to be away. In that case, can you file an issue for this and open up a new PR for it? Then let's merge this PR. Thanks!